### PR TITLE
Feature/add opening times

### PR DESCRIPTION
**Pull request by markysoft**
Added reception and surgery opening times

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,56 @@ The output JSON will be an arary of objects in the following format:
...
+            "tuesday" : [ 
+                {
+                    "opens" : "08:00",
+                    "closes" : "13:00"
+                }, 
+                {
+                    "opens" : "15:00",
+                    "closes" : "19:00"
+                }
+            ...
```
Comment (st3v3nhunt) [inline README.md]: Like it 👍

File: app/lib/mappers/mapOpeningTimes.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+function getSession(session) {
+  return {
+    opens: session['s:fromTime'],
+    closes: session['s:toTime'],
+  };
+}
+
+function getSessions(session) {
+  const timesSessions = session['s:timesSession'];
+  if (timesSessions.constructor === Array) {
```
Comment (st3v3nhunt) [inline app/lib/mappers/mapOpeningTimes.js]: Is `timesSessions` ever going to be `null`/`undefined`?

File: app/lib/mappers/mapOpeningTimes.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    return [];
+  }
+  return [
+    getSession(timesSessions),
+  ];
+}
+
+function mapOpeningTimes(rawTimes) {
+  const openingTimes = {};
+  rawTimes['s:daysOfWeek']['s:dayOfWeek'].forEach((dayOfWeek) => {
```
Comment (st3v3nhunt) [inline app/lib/mappers/mapOpeningTimes.js]: Are either `rawTimes['s:daysOfWeek']` or `['s:dayOfWeek']` ever going to be `null`/`undefined`?

File: app/lib/mappers/mapOpeningTimes.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    openingTimes[dayOfWeek['s:dayName'].toLowerCase()] = getSessions(dayOfWeek['s:timesSessions']);
```
Comment (st3v3nhunt) [inline app/lib/mappers/mapOpeningTimes.js]: Will either `dayOfWeek['s:dayName']` or `dayOfWeek['s:timesSessions']` ever be `null`/`undefined`?

File: test/resources/overview-no-opening-times.json [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+                        "s:performance": {
+                            "s:url": "http://www.nhs.uk/Services/GP/Performance/DefaultView.aspx?id=36931",
+                            "s:text": "More on how we perform"
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (st3v3nhunt) [inline test/resources/overview-no-opening-times.json]: New line please

**Comment by markysoft** [comment]
Added null checks, moved full json example out of README.mds into a separate file.
Add rate limiter to set overview scans to 5000 an hour. Reduced workers to 1.
Moved integration tests out of git hub and put them on travis' after_success

**Pull request merged**